src/core/sensor/qgsiodevicesensor.cpp

```diff
@@ -353,10 +353,46 @@ void QgsSerialPortSensor::setPortName( const QString &portName )
   mPortName = portName;
 }
 
+// by jtornero
```

Comment:
Please take this out too, it's not something we do in our source files, we let people discover this info via git blame :wink: